lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -294,27 +295,30 @@ def test_user_doesnotexist(self):
         self.request.is_ajax.return_value = False
         resp = view_user_doesnotexist(self.request)  # pylint: disable=assignment-from-no-return
         self.assertEqual(resp.status_code, 400)
-        self.assertIn("User does not exist", resp.content)
+        self.assertIn("User does not exist", resp.content.decode("utf-8"))
 
     def test_user_doesnotexist_ajax(self):
         self.request.is_ajax.return_value = True
         resp = view_user_doesnotexist(self.request)  # pylint: disable=assignment-from-no-return
         self.assertEqual(resp.status_code, 400)
-        self.assertIn("User does not exist", resp.content)
+        self.assertIn("User does not exist", resp.content.decode("utf-8"))
 
     @ddt.data(True, False)
     def test_alreadyrunningerror(self, is_ajax):
         self.request.is_ajax.return_value = is_ajax
         resp = view_alreadyrunningerror(self.request)  # pylint: disable=assignment-from-no-return
         self.assertEqual(resp.status_code, 400)
-        self.assertIn("Requested task is already running", resp.content)
+        self.assertIn("Requested task is already running", resp.content.decode("utf-8"))
 
     @ddt.data(True, False)
     def test_alreadyrunningerror_with_unicode(self, is_ajax):
         self.request.is_ajax.return_value = is_ajax
         resp = view_alreadyrunningerror_unicode(self.request)  # pylint: disable=assignment-from-no-return
         self.assertEqual(resp.status_code, 400)
-        self.assertIn('Text with unicode chárácters', resp.content)
+        self.assertIn(
+            'Text with unicode chárácters',
+            resp.content if six.PY2 else resp.content.decode('utf-8')
```

Comment:
Why can't we just always run decode here on the content as we dod with other tests?

Reply:
@feanil I will debug this again and share the findings.

Reply:
@feanil Please check.